src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3532,6 +3383,244 @@
   <data name="AddPage" xml:space="preserve">
     <value>Add page</value>
   </data>
+  <data name="ProcessingItems" xml:space="preserve">
+    <value>Processing items...</value>
+  </data>
+  <data name="SpeedWithColon" xml:space="preserve">
+    <value>Speed:</value>
+  </data>
+  <data name="StatusCenter_CompressCanceled_Header" xml:space="preserve">
+    <value>Canceled - compressing {0} items to "{1}"</value>
+    <comment>Shown in a StatusCenter card.</comment>
+  </data>
+  <data name="StatusCenter_CompressCanceled_SubHeader" xml:space="preserve">
+    <value>Canceled compressing {0} items from {1} to {2}</value>
+    <comment>Shown in a StatusCenter card.</comment>
+  </data>
```

Comment:
Canceling compression only needs a single header "Canceled creating archive.zip".

Reply:
Another option is
"Canceled compressing {0} items"
"Canceled compressing {0} items from {1} to {2}."